src/main/java/org/jboss/wise/shared/GWTClientConversationBean.java

```diff
@@ -288,6 +289,15 @@ private TreeElement wiseOutputTransfer(WiseTreeElement wte) {
             gTreeElement.setProtoType(pElement);
          }
 
+         Type[] typeArr = ((ParameterizedType)wte.getClassType()).getActualTypeArguments();
+         if (typeArr != null && typeArr.length > 0) {
+            String actualType = typeArr[0].toString();
+            gTreeElement.setRawType(actualType);
+         } else {
+            System.out.println("ERROR parameterizedType actualTypeArguments not found for "
```

Comment:
We should use proper logger instead of System.out (assuming this is not something meant to be returned as error to the user interface).